app/src/index.jsx

```diff
@@ -9,6 +9,12 @@ import "react-redux-toastr/lib/css/react-redux-toastr.min.css";
 import store from "./redux/store";
 import App from "./app";
 
+window.addEventListener("vite:preloadError", (event) => {
+  event.preventDefault();
```

Comment:
le preventDefault est vraiment nécessaire, si on reload tout s'arrete de toute façon ? 
la doc ne l'indique pas specialement  https://vitejs.dev/guide/build#load-error-handling

Reply:
Je sais pas si c'est utile effectivement. Ils disent juste "If you call event.preventDefault(), the error will not be thrown."